scripts/bountiful.ash

```diff
@@ -1,15 +1,14 @@
 /******************************************************************************
-* Bountiful by RESPRiT
-* Version 0.3
-* https://github.com/RESPRiT
+* Bountiful by loathers
+* https://github.com/loathers
 *
-* Adapted from AutoBHH:
-* -Originally by izchak
-* -Major revisions by Raorn and Zarqon
+* Forked from RESPRiT
+* -Which was adapted from AutoBHH:
+*  -Originally by izchak
+*  -Major revisions by Raorn and Zarqon
 ******************************************************************************/
 script "Bountiful";
-notify tamedtheturtle;
-since r18000;
+since r26686;
```

Comment:
let's do r26691 for this, since that's when gausie added the support for git install w/o the full URL